buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -63,6 +67,10 @@ object Config {
         // Gradle version plugin; use dependencyUpdates task to view third party dependency updates via `./gradlew dependencyUpdates` or AS Gradle -> [project]] -> Tasks -> help -> dependencyUpdates
         // https://github.com/ben-manes/gradle-versions-plugin/releases
         const val GRADLE_VERSIONS = "com.github.ben-manes:gradle-versions-plugin:0.41.0"
+
+        // Gradle plugin and a core android library thought to run screenshot tests for Android
+        // https://github.com/pedrovgs/Shot
+        const val KARUMI_SCREENSHOT_TESTING = "com.karumi:shot:5.14.0"
```

Comment:
@Br-DiorD nit: Update to the latest released versions, `5.14.1` (see https://github.com/pedrovgs/Shot/releases)

Reply:
@br-jared-stockton I downgraded to 5.14.0 due to an issue they were still trying to fix with 5.14.1. I'm trying to figure out what it was now.

Reply:
nvm appears fixed whatever it was yay !